### ISSUE TITLE
Unable to extract info section (vimeo)

### ISSUE THREAD

**Issue by austux** [rg3/youtube-dl#17730]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.09.26*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2018.09.26**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [X] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add the `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

[this worked immediately beforehand (and does now, same provider): youtube-dl -f http-720p https://player.vimeo.com/video/291018673]

```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-vf', u'http-720p', u'https://player.vimeo.com/video/291221781']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2018.09.26
[debug] Python version 2.7.15rc1 (CPython) - Linux-4.15.0-35-generic-x86_64-with-Ubuntu-18.04-bionic
[debug] exe versions: exe versions: ffmpeg 3.4.4-0ubuntu0.18.04.1, ffprobe 3.4.4-0ubuntu0.18.04.1, rtmpdump 2.4
[debug] Proxy map: {}
[vimeo] 291030714: Downloading webpage
[vimeo] 291030714: Extracting information
ERROR: Unable to extract info section (caused by RegexNotFoundError(u'Unable to extract \x1b[0;34minfo section\x1b[0m; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 555, in _real_extract
    flags=re.DOTALL)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 977, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract info section; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 792, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 507, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 569, in _real_extract
    cause=e)
ExtractorError: Unable to extract info section (caused by RegexNotFoundError(u'Unable to extract \x1b[0;34minfo section\x1b[0m; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
<end of log>
```
[fails likewise with https://player.vimeo.com/video/291030714]

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://www.youtube.com/watch?v=BaW_jenozKc
- Single video: https://youtu.be/BaW_jenozKc
- Playlist: https://www.youtube.com/playlist?list=PL4lCao7KL_QFVb7Iudeipvc2BCavECqzc

Note that **youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. In order for site support request to be accepted all provided example URLs should not violate any copyrights.

---

### Description of your *issue*, suggested solution and other information

three of four video downloads fails with a "RegexNotFoundError(u'Unable to extract info section" error; the download window for these videos is time-limited to about 18 hours from now.

those downloads are succeeding with the Video DownloadHelper 7.3.5 [Built on Wed Aug 01 2018 16:23:45 GMT+0200 (CEST)] plugin for FireFox, however the filenames are being somewhat truncated (minor, I know). I strongly prefer to semi-automate video downloads (large numbers or long waiting time) with youtube-dl when possible.

**Comment by austux** [comment]
two Video DownloadHelper streams are running in parallel now, so this is evidently not a parallelism-thwarting measure by the server.

**Comment by austux** [comment]
it is, however, not so simple.

downloading at 1280x720 (highest resolution advertised) fails (no moov atom in resulting MP4 file).

other resolutions advertised are 960x540 and 640x360. now attempting those.

**Comment by austux** [comment]
no joy there, either, yet the videos play just fine in FireFox

**Comment by austux** [comment]
apparently, there is a work-around: https://en.savefrom.net/ plus the browser plugin mentioned there pulls down 1280x720 MP4s of the three recalcitrant video clips.

as at now, I have no idea what the key difference between the three systems is.

**Comment by kaovilai** [comment]
So I have the same issue with https://player.vimeo.com/video/293644418

@dstftw  If this is duplicate where can I find the non duplicate issue page?

Thanks.

**Comment by mikhailmakarov** [comment]
I have the same issue with https://player.vimeo.com/video/294575256

**Comment by 414n** [comment]
@kaovilai  @mikhailmakarov Had the same issue too, but solved it using 2018.10.05 version.
